packages/ckeditor5-markdown-gfm/docs/_snippets/features/markdown.html

```diff
@@ -1,4 +1,24 @@
-<div id="snippet-markdown">This is [CKEditor 5](https://ckeditor.com).</div>
+<div id="snippet-markdown">
+
+<h2>The Markdown language</h2>
+<p>Markdown is a lightweight markup language that you can use to add formatting elements to plaintext text documents. Created by <a href="https://daringfireball.net/projects/markdown/" target="_blank">John Gruber</a> in 2004, Markdown is now one of the world&rsquo;s most popular markup languages.</p>
+
+<h2>Editing with markdown output</h2>
+<p>The <a href="https://ckeditor.com">CKEditor 5 WYSIWYG</a> editor lets the user utilize this flexible yet simple markup language, in the GitHub flavor. The output covers the most important features, like <a href="https://ckeditor.com">links</a>, <strong>different</strong> kinds of <em>emphasis</em>, the <code>inline code formatting</code> or the code blocks:</p>
+
+<pre>
+<code class="language-css">
+	p {
+		text-align: center;
+		color: red;
+	  }
```

Comment:
Actually, I think it will display better if we remove the indent, so it won't have an unnecessary indentation in the code block.

```suggestion
p {
  text-align: center;
  color: red;
}
```